### ISSUE TITLE
Can't install 5.19.6~7

### ISSUE THREAD

**Issue by choiman1559** [bkw777/mainline#155]
```
~$ sudo mainline --install 5.19.7
[sudo]  Password: 
mainline 1.0.16
Distribution: Ubuntu 20.04.5 LTS
Architecture: amd64
Running kernel: 5.19.2-051902-generic
Fetching index from kernel.ubuntu.com...
OK

Fetching index...

▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ 100 %  

----------------------------------------------------------------------
Found installed: 5.19.2-051902.202208171037
Found installed: 5.4.0-81.91
----------------------------------------------------------------------
----------------------------------------------------------------------
Preparing to install '5.19.7'
dpkg: error: --install option requires at least one package archive file argument.

Type dpkg --help for help about installing and deinstalling packages [*];
Use 'apt' or 'aptitude' for user-friendly package management;
Type dpkg -Dhelp for a list of dpkg debug flag values;
Type dpkg --force-help for a list of forcing options;
Type dpkg-deb --help for help about manipulating *.deb files;

Options marked [*] produce a lot of output - pipe it through 'less' or 'more' !
E: Installation completed with errors

```

5.19.6~7 is not displayed in the UI, and even if I try on the console, it does not install as above.

Seeing that there is an invalid file in the cache folder for that version, it's probably a problem with the mainline's html parsing.

**Comment by bkw777** [comment]
5.19.6 and .7 don't exist

**Comment by choiman1559** [comment]
That's not true.  See [kernel.org](kernel.org) It has already been released up to [5.19.8](https://cdn.kernel.org/pub/linux/kernel/v5.x/ChangeLog-5.19.8).  (As of September 9)
